services/web/server/src/simcore_service_webserver/catalog_handlers.py

```diff
@@ -13,6 +13,7 @@
 )
 from pint import UnitRegistry
 from pydantic import ValidationError
+from servicelib.request_keys import RQT_USERID_KEY
```

Comment:
TIP: from my PR, all the constants are exposed as well in ``_constant`` w/o risk of cross-dependencies

Reply:
Ok thanks, I will wait after you merge it, so i add extend it and add there others:
from servicelib.rest_constants import RESPONSE_MODEL_POLICY